src/main/main.js

```diff
@@ -887,16 +889,25 @@ function handleUpdateMenuEvent(event, configData) {
   }
 }
 
+async function handleSelectDownload(event, startFrom) {
+  const message = 'Specify the folder where files will download';
+  const result = await dialog.showOpenDialog({defaultPath: startFrom,
+    message,
```

Comment:
Nit: do you have eslint enabled? I feel like in a few places ident and spacing is not matching the settings in the webapp, maybe I am not seeing correctly tho :D

Reply:
Indenting in the desktop app has always been 2 spaces for some reason. We should fix that at some point.